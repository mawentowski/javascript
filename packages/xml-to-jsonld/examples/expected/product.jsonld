{
  "@context": "https://schema.org",
  "@type": "Product",
  "name": "Example Widget",
  "description": "A versatile widget for demonstration purposes.",
  "image": [
    "https://example.com/widget.jpg"
  ],
  "brand": {
    "@type": "Brand",
    "name": "ExampleCo"
  },
  "sku": "WIDGET-001",
  "offers": [
    {
      "@type": "Offer",
      "price": 19.99,
      "priceCurrency": "USD",
      "availability": "InStock"
    }
  ]
}