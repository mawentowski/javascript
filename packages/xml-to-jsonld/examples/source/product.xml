<!-- <?xml-model href="../packages/schemas/rnc/site-seo.rnc"
            type="application/relax-ng-compact-syntax"?> -->
<Product>
  <name>Example Widget</name>
  <description>A versatile widget for demonstration purposes.</description>
  <image>
    <ImageObject>
      <url>https://example.com/widget.jpg</url>
    </ImageObject>
  </image>
  <sku>WIDGET-001</sku>
  <brand>
    <Brand>
      <name>ExampleCo</name>
    </Brand>
  </brand>
  <offers>
    <Offer>
      <priceCurrency>USD</priceCurrency>
      <price>19.99</price>
      <availability>InStock</availability>
      <url>https://example.com/widget</url>
    </Offer>
  </offers>
</Product>