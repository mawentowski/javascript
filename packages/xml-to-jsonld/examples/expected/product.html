<article>
    <h1>Example Widget</h1>
    <p>A versatile widget for demonstration purposes.</p>
    <p><strong>From:</strong> 19.99 USD</p>
  </article>